src/selectors/visibleColumnBreakpoints.js

```diff
@@ -101,10 +102,20 @@ export function getColumnBreakpoints(pausePoints, breakpoints, viewport) {
     ({ location: { line } }) => lineCount[line] > 1
   );
 
-  return columnBreakpoints.map(({ location }) => ({
-    location,
-    enabled: isEnabled(location, breakpointMap)
-  }));
+  return columnBreakpoints.map(({ location }) => {
+    const foundBreakpoint = breakpoints.find(breakpoint => {
+      return (
+        breakpoint.location.line === location.line &&
+        breakpoint.location.column === location.column
+      );
+    });
+
+    return {
+      location,
+      enabled: isEnabled(location, breakpointMap),
```

Comment:
isn't isEnabled trying to find the breakpoint too?

can we re-use this?

Reply:
Good shout; we'll be able to move to `.get` instead of `hasIn` when https://github.com/devtools-html/debugger.html/pull/7366 lands.